identity-account/README.md

```diff
@@ -0,0 +1,70 @@
+# Iota Identity Account
+
+The Account is an interface for creating and managing identities on the Iota Tangle. It includes easy-to-use functions that allow many operations including:
+
+- Creating an identity or resolving an existing DID Document.
+- Updating a DID Document by adding or removing:
+  - Services.
+  - Verification Methods.
+  - Verification Relationships.
+  - Custom properties.
+- Managing and storing the state of DID Documents.
+- Managing and storing private encryption keys used as verification methods.
+
+## Account Creation
+
+Creating an account is done through the `AccountBuilder`.
+
+```rust
+  let account: Account = Account::builder()
+    .create_identity(IdentitySetup::default())
+    .await?;
+```
+
+Various options can be set for the `AccountBuilder` including:
+
+- `autosave`: sets the account auto-save behaviour.
+  - `AutoSave::Every` => save to storage on every update.
+  - `AutoSave::Never` => never save to storage when updating.
+  - `AutoSave::Batch` => save to storage after every `n` updates.
+- `autopublish`: sets the account auto-publish behaviour.
+  - `true` => publish to the Tangle on every DID document change.
+  - `false` => never publish automatically.
+- `storage`: sets the account storage adapter.
+- `client`: sets the IOTA Tangle `Client`, this determines the `Network` used by the identity.
+- `create_identity`: creates a new identity based on the builder configuration and returns an `Account` instance to manage it.
+- `load_identity`: loads an existing identity with the specified `did` using the current builder configuration. The identity must exist in the configured `Storage`.
```

Comment:
Try not to duplicate code definitions, use intradoc links at most. Or just include a code example similar to the `config` example to demonstrate some of the available options.

```suggestion
```

Reply:
It's clear it has maintainability issues, but It's either duplicating the examples or duplicating the function definitions, I think having the code definitions in the README gives a quick overview about the possible options that can be done with the account.

Reply:
We have the proper examples for a reason.

They can similarly see the range of functions if they click on the `Account` intradoc link.

The only thing obscure is really the update functions, so those should probably be linked to via `IdentityUpdater`.